packages/mui-codemod/src/deprecations/badge-props/badge-props.test.js

```diff
@@ -0,0 +1,53 @@
+import path from 'path';
+import { expect } from 'chai';
+import transform from './badge-props';
+import { jscodeshift } from '../../../testUtils';
+import readFile from '../../util/readFile';
+
+function read(fileName) {
+  return readFile(path.join(__dirname, fileName));
+}
+
+describe('@mui/codemod', () => {
+  describe('deprecations', () => {
+    describe('badge-props', () => {
+      it('transforms props as needed', () => {
+        const actual = transform({ source: read('./test-cases/actual.js') }, { jscodeshift }, {});
```

Comment:
```suggestion
        const actual = transform({ source: read('./test-cases/actual.js') }, { jscodeshift }, { printOptions: { trailingComma: true } });
```

Reply:
these is missing in slider any specific reason why?